packages/cli/src/util/runner.ts

```diff
@@ -0,0 +1,38 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { IPrismHttpServer } from '@stoplight/prism-http-server/src/types';
+import * as chokidar from 'chokidar';
+import { CreatePrismOptions } from './createServer';
+
+export function runPrism(createPrism: Function, options: CreatePrismOptions, spec: string) {
+  return createPrism(options).then((server: IPrismHttpServer) => {
+    if (server) {
+      const watcher = chokidar.watch(spec, { awaitWriteFinish: { stabilityThreshold: 500, pollInterval: 100 } });
+
+      watcher.on('change', () => {
+        server.fastify.log.info('Restarting Prism...');
```

Comment:
Do not use the logger in the fastify server — this one shouldn't be touched at all. Just import `signale` in this file and log the things directly.